java/src/main/java/org/rocksdb/ExportImportFilesMetaData.java

```diff
@@ -0,0 +1,23 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * The metadata that describes a column family.
+ */
+public class ExportImportFilesMetaData extends RocksObject {
+  private ExportImportFilesMetaData(final long nativeHandle) {
+    super(nativeHandle);
+    // We do not own the native object!
+    disOwnNativeHandle();
```

Comment:
@mayuehappy can you explain why you are callng `disOwnNativeHandle` here please?

Reply:
@adamretter Thanks for the reviewing, I thought about this place again. `org.rocksdb.ExportImportFilesMetaData` should indeed own `ROCKSDB_NAMESPACE::ExportImportFilesMetaData`'s nativeHandle. My initial idea was a little wrong, so I deleted these two lines of code.